lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -511,16 +511,16 @@ sub create {
 
     $edit->insert;
 
-    my $now = DateTime->now;
     my $duration = DateTime::Duration->new( days => $conditions->{duration} );
+    my $interval = DateTime::Format::Pg->format_interval($duration);
 
     my $row = {
         editor => $edit->editor_id,
         data => JSON::Any->new( utf8 => 1 )->objToJson($edit->to_hash),
         status => $edit->status,
         type => $edit->edit_type,
-        open_time => $now,
-        expire_time => $now + $duration,
+        open_time => \"now()",
+        expire_time => \"now() + interval '$interval'",
```

Comment:
See 	01d2ac0. I noticed that locally, auto-edits were "closing" 6 hours after they were opened (I'm in UTC-6), so I couldn't even test this ticket without fixing this first.